openapi.json

```diff
@@ -3269,6 +3263,78 @@
           "identityStatus": "VERIFIED",
           "createDate": "2022-07-01T20:29:23Z"
         }
+      },
+      "TranscribeUtterance": {
+        "title": "TranscribeUtterance",
+        "x-stoplight": {
+          "id": "u509r3zqc38ze"
+        },
+        "type": "object",
+        "description": "The `TranscribeUtterance` command transcribes the caller’s voice and returns transcription of the audio and optionally returns the recording of the audio transcribed.\n\n`TranscribeUtterance` is blocking and is a terminal command. As such, the actionUrl property is required, and control of the Call picks up using the `PerCL` returned in response of the `actionUrl`. Recording and Transcription information is returned in the actionUrl request. If the reason this command ended was due to the call hanging up, any PerCL returned will not execute.",
+        "x-percl-command": true,
+        "properties": {
+          "actionUrl": {
+            "type": "string"
+          },
+          "playBeep": {
+            "type": "boolean",
+            "default": false
+          },
+          "record": {
+            "type": "object",
+            "properties": {
+              "saveRecording": {
+                "type": "boolean",
+                "default": false
+              },
+              "maxLengthSec": {
+                "type": "integer",
+                "minimum": 1,
+                "maximum": 60,
+                "default": 60
+              },
+              "rcrdTerminationSilenceTimeMs": {
+                "type": "integer",
+                "exclusiveMinimum": true,
+                "minimum": 0,
+                "maximum": 3000
+              }
+            }
+          },
+          "privacyForLogging": {
+            "type": "boolean",
+            "default": false
+          },
+          "privacyForRecording": {
+            "type": "boolean",
+            "default": false
+          },
+          "prompts": {
+            "type": "array",
+            "items": {}
+          }
+        },
+        "required": ["actionUrl"],
+        "x-examples": {
+          "Example 1": {
+            "TranscribeUtterance": {
+              "actionUrl": "http://foo.vail/handleTranscription",
+              "record": {
+                "maxLengthSec": 60,
+                "saveRecording": true,
+                "rcrdTerminationSilenceTimeMs": 2000
+              },
+              "playBeep": true,
+              "prompts": [
+                {
+                  "Play": {
+                    "file": "http://dplutil01.vail/~user/prompts/please-record-greeting.wav"
+                  }
+                }
+              ]
+            }
+          }
+        }
```

Comment:
Why is this appearing as a diff in this way? shouldn't there just be an `x-percl-command` update? Also it shouldn't be removing properties unless that was an intended change.

Reply:
This is an intended change, Logan had removed the x-max-lengths properties and this openapi.json file did not have this change, so once autogenerated, it applied the openapi changes from the latest branch to this file